hack/get-git-dirty-postfix.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
```

Comment:
Why do we need a dedicated script for this while

Reply:
Hmm, I'm no expert in Makefiles. I didn't work, when I included it in the Makefile.